rules/use-github-discussions/rule.md

```diff
@@ -15,33 +15,73 @@ related: []
+### Stage 1 - GitHub Discussions
+
+GitHub Discussions is the right place to start when **certainty** is low. Raising it in this setting lets the whole team discuss the proposal and reach a consensus about what to do. Make sure to @mention any important stakeholders e.g. @BobNorthwind
 
 ![Figure: SSW Rules | GitHub | Discussions](https://user-images.githubusercontent.com/66365977/214453285-b074f967-a637-4968-bd0d-ce79198f8bc3.png)
 
-From there you can add your comments and let others know about it by @ mentioning them (E.g. @BobNorthwind). When you're done discussing and you have something to action on the project, you can then easily convert the discussion into an issue!
+### Stage 2 - GitHub Issues
+
+Once the team has made a decision on a GitHub Discussion or if you are certain the issue doesn't need a discussion, then you need to evaluate the **difficulty** of implementation. If it is **difficult** to implement, then GitHub Issues is the right place to put it since it provides a place to manage work as the improvement is developed.
+
+GitHub Discussions can be converted to GitHub Issues with a single click! 
 
 ![Figure: SSW Rules | GitHub | Discussions - Convert a Discussion to an issue](https://user-images.githubusercontent.com/66365977/214461325-7bd47032-3b8f-4e1c-96f4-63b7ee02a64d.png)
 
 ![Figure: SSW Rules Content Discussion Converted To Issue](https://user-images.githubusercontent.com/66365977/214463615-2b27e427-93a9-4d49-ab2b-bdb06859c816.png)
 
+### Stage 3 - Pull Request
+Once the work is complete or if it is an **easy** change you are **certain** you want to make (e.g. a spelling mistake), it can be turned into a Pull Request.
+
+## Customer support
+GitHub Discussions provide a great forum for users to ask questions about how to use the product. Not only is it easy to access, but it will also help future users with the same problem.
+
+## Technical discussions
+If you need help with a problem, and Stack Overflow has failed you, then GitHub Discussions offers a way to relay with your team.
+
+## Gauging community sentiment
+Community engagement is hard to achieve but can provide invaluable insights into contentious parts of the application. A post in GitHub Discussions lets you quickly poll all the users of your app.
+
+# Categorisation
+GitHub Discussions allow you to set up categories. Getting these going is important to help users sift through the different topics. Keep the categories simple and easy to follow so that anyone who jumps in will know where to begin.
 
-Now that it's an issue, you can of course associate Pull Requests with that issue and otherwise associate work with that rather than the whole project.
+![Figure: Good Example - TinaCMS is an example of a project with well categories and active GitHub Discussions](TinaCMSDiscussions.png)
```

Comment:
All the other GitHub screenshots are in dark mode.
1. Please make them consistent